schema.json

```diff
@@ -0,0 +1,514 @@
+    "statements": {
+      "type": "array",
+      "items": {
+        "anyOf": [
+          {
+            "$ref": "#/$defs/forEach"
+          },
+          {
+            "$ref": "#/$defs/if"
+          },
+          {
+            "$ref": "#/$defs/switch"
+          },
+          {
+            "$ref": "#/$defs/invocation"
+          },
+          {
+            "$ref": "#/$defs/assignment"
+          },
+          {
+            "$ref": "#/$defs/return"
+          }
+        ]
+      }
+    },
+    "forEach": {
+      "type": "object",
+      "properties": {
+        "$type": {
+          "const": "LivingDocumentation.ForEach, LivingDocumentation.Statements"
+        },
+        "Expression": {
+          "type": "string"
+        },
+        "Statements": {
+          "$ref": "#/$defs/statements"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "Expression"
+      ]
+    },
+    "if": {
+      "type": "object",
+      "properties": {
+        "$type": {
+          "const": "LivingDocumentation.If, LivingDocumentation.Statements"
+        },
+        "Sections": {
+          "type": "array",
+          "items": {
+            "Condition": {
+              "type": "string"
+            },
+            "Statements": {
+              "$ref": "#/$defs/statements"
+            }
+          }
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "Sections"
+      ]
+    },
+    "switch": {
+      "type": "object",
+      "properties": {
+        "$type": {
+          "const": "LivingDocumentation.Switch, LivingDocumentation.Statements"
+        },
+        "Expression": {
+          "type": "string"
+        },
+        "Sections": {
+          "type": "array",
+          "items": {
+            "Labels": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "Statements": {
+              "$ref": "#/$defs/statements"
+            }
+          }
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "Expression"
+      ]
+    },
+    "invocation": {
+      "type": "object",
+      "properties": {
+        "$type": {
+          "const": "LivingDocumentation.InvocationDescription, LivingDocumentation.Descriptions"
+        },
+        "ContainingType": {
+          "type": "string"
+        },
+        "Name": {
+          "type": "string"
+        },
+        "Arguments": {
+          "$ref": "#/$defs/arguments"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "ContainingType",
+        "Name"
+      ]
+    },
+    "assignment": {
+      "type": "object",
+      "properties": {
+        "$type": {
+          "const": "LivingDocumentation.AssignmentDescription, LivingDocumentation.Descriptions"
+        },
+        "Left": {
+          "type": "string"
+        },
+        "Operator": {
+          "type": "string"
```

Comment:
Do we know which strings this can be, that is, could this be an `"enum": [...]`?

Reply:
That makes sense.